presto-hive-metastore/src/main/java/com/facebook/presto/hive/metastore/MetastoreUtil.java

```diff
@@ -502,6 +502,9 @@ public static FileSystem getFileSystem(HdfsEnvironment hdfsEnvironment, HdfsCont
         try {
             return hdfsEnvironment.getFileSystem(context, path);
         }
+        catch (PrestoException e) {
+            throw e;
+        }
```

Comment:
these are redundant since you are catching and throwing back immediately.

Reply:
you are right . will remove this :neutral_face: